reto.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Recursion do
+  def print_multiple_times(msg, cantidad, cont, limit_inf, limit_sup) when cont <= limit_sup do
+    msg = msg <> "#{cont}"
+    numero = String.to_integer(msg)
+    if cont >= limit_inf do
+      IO.puts(numero)
+      cond do
+      rem(numero,3) == 0 -> print_multiple_times(msg, cantidad + 1, cont + 1, limit_inf, limit_sup)
+      true -> print_multiple_times(msg, cantidad, cont + 1, limit_inf, limit_sup)
+      end
+    else
+      print_multiple_times(msg, cantidad, cont + 1, limit_inf, limit_sup)
+    end
+  end
+
+  def print_multiple_times(_msg,cantidad, cont, _limit_inf, limit_sup ) when cont > limit_sup do
+      cantidad
+  end
+end
+
+numeros =  Recursion.print_multiple_times("",0, 1, 3, 5)
```

Comment:
Numeros/numbers is not a proper name for the result, it could be something like e.g divisible_numbers_amount or simply "result", what do you think about this?